sections/k-partners-listing.liquid

```diff
@@ -0,0 +1,471 @@
+{{ 'k-partners-listing.css' | asset_url | stylesheet_tag }}
+
+{%- style -%}
+    #shopify-section-{{ section.id }} {
+        --buttons-radius-outset: 40px;
+        --buttons-radius: 40px;
+        z-index: 2;
+        position: relative;
+    }
+    .section-{{ section.id }}-padding {
+        padding-top: {{ section.settings.padding_top | times: 0.75 | round: 0 }}px;
+        padding-bottom: {{ section.settings.padding_bottom | times: 0.75 | round: 0 }}px;
+    }
+
+    @media screen and (min-width: 750px) {
+        .section-{{ section.id }}-padding {
+        padding-top: {{ section.settings.padding_top }}px;
+        padding-bottom: {{ section.settings.padding_bottom }}px;
+        }
+    }
+{%- endstyle -%}
+{% assign page_country = section.settings.country %}
+{% assign us_state = page.metafields.custom['country_reference'].value.is_us_state %}
+
+{% if us_state == true %}
+    {% assign page_country = page_country | append: ', USA' %}
+{% endif %}
+
+<div class="page-width section-{{ section.id }}-padding isolate{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}" data-cascade>
+    {%- unless section.settings.title == blank -%}
+        <div class="title-wrapper-with-link title-wrapper--self-padded-mobile title-wrapper--no-top-margin">
+            <h1 class="title main-title inline-richtext {{ section.settings.heading_size }}">
+            {{ section.settings.title | replace: '[country]', page_country }}
+            </h1>
+        </div>
+    {%- endunless -%}
```

Comment:
I think this top heading is unnecessary and can be removed. We already have two headings below it with the area name.